### PR TITLE
created form and make it works

### PR DESCRIPTION
**Pull request by Fbouchezpro**

### CODE REVIEW

**Review by Fostrankill — DISMISSED** [review]

File: frontend/src/components/Form/style.css [hunk 1]
```diff
@@ -130,4 +131,4 @@ button.form-button {
   width: 100vw;
   height: 6rem;
   grid-area: 5 / 1 / 6 / 4;
-}
+} */
```
Comment (LoicBrassart) [inline frontend/src/components/Form/style.css]: Dead code

File: frontend/src/components/Formradio/style.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  background-color: #111111;
+  gap: 0.5rem;
+  height: 60vh;
+  padding-top: 1rem;
+}
+.containerForm {
+  /* display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(5, 1fr);
+  grid-row-gap: 3rem; */
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/style.css]: Dead code

File: frontend/src/components/Formradio/style.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+  padding-top: 4rem;
+  width: 100vw;
+  color: #daa622;
+}
+
+.genre {
+  /* grid-area: 1 / 1 / 2 / 4; */
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/style.css]: Same

File: frontend/src/components/Formradio/style.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  /* background-image: url("../../assets/genre.svg"); */
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/style.css]: Same

File: frontend/src/components/Formradio/style.css [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+  /* background-repeat: no-repeat;
+  background-position: center; */
+  /* background-size: 256px; */
+  justify-content: space-around;
+  font-size: 2rem;
+  font-family: Bangers;
+  text-align: center;
+  /* padding-top: 2.6rem;
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/style.css]: Same

File: frontend/src/components/Formradio/style.css [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+  padding-right: 1.5rem; */
+}
+
+label[for="1"] {
+  content: url("../../assets/action.svg");
+  /* align-self: center;
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/style.css]: Same

File: frontend/src/components/Formradio/style.css [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+  position: relative;
+  left: 1.5rem; */
+  cursor: pointer;
+}
+label[for="2"] {
+  content: url("../../assets/sci-fi.svg");
+  /* align-self: center;
```
Comment (LoicBrassart) [inline frontend/src/components/Formradio/style.css]: Etc, etc...

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove dead code -_-

**Comment by LoicBrassart** [comment]
Mr @Fostrankill , please **read** your PRs: there is less than 2 minutes between the PR creation and your review, and you "missed" whole blocks of code è_é

**Review by Fostrankill — APPROVED** [review]
Commentary code were deleted.

**Review by LoicBrassart — APPROVED** [review]
No more Evil that I saw

**Pull request merged**